src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements.mdx

```diff
@@ -9,12 +9,32 @@ redirects:
+The [Kubernetes integration](/docs/integrations/kubernetes-integration/get-started/introduction-kubernetes-integration) is compatible with many different platforms including GKE, EKS, AKS, OpenShift, and more. Each has a different compatibility with our integration. You can find more information in this page.
+
+## Requirements [#reqs]
+
+The New Relic Kubernetes integration has the following requirements:
+
+* A New Relic account. Don't have one? [Sign up for free](https://newrelic.com/signup). No credit card required.
+* Linux distribution [compatible with New Relic infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure#operating-systems).
+* [`kube-state-metrics`](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-state-metrics) running on the cluster.
+* When using CRI-O as the container runtime, the [processes](/attribute-dictionary/?event=ProcessSample) inside containers are not reported. Performance data is collected at the [container](/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data#container-data) level.
+* If you are using Helm, see the specific [compatibility and requirements](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#compatibility). 
 
 <Callout variant="important">
-If you are using Openshift, you can also use `kubectl` most of the time but be careful that `kubectl` does not have commands like `oc login` or `oc adm`. You may need to use `oc` instead of `kubectl`.
+  `kube-state-metrics` v2 or higher is supported from integration version 3.6.0 or higher.
+  <br/>Install the Kubernetes integration up to version 3.5.0 if you are using `kube-state-metrics` 1.9.8 or lower.
+  <br/>
+  Check the `values.yaml` file if you're updating `kube-state-metrics` from v1.9.8 to v2 or higher because some variables may have changed:
+
+  - [KSM v1.9.8](https://github.com/kubernetes/kube-state-metrics/blob/0f5e4969f24bfa5f7c884f66d9d6fdf6f1921c16/charts/kube-state-metrics/values.yaml)
+  - [KSM v2 or higher](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-state-metrics/values.yaml)
 </Callout>
 
+## Container information [#containers]
+
+Our Kubernetes integration is [CRI](https://kubernetes.io/docs/concepts/architecture/cri)-agnostic. It's been specifically tested to be compatible with Containerd, CRI-O, and Docker.
+
 ## Compatibility [#compatibility]
 
 Our Kubernetes is compatible and is continuously tested on the following versions:
```

Comment:
Probably out of the scope of this PR, but having a table for a single row seems a bit strange.

Reply:
I've removed the table.